packages/arb-token-bridge-ui/src/util/RetryableUtils.ts

```diff
@@ -1,23 +1,26 @@
 import {
   L1TransactionReceipt,
-  L1ToL2MessageWriter as IL1ToL2MessageWriter
+  L1ToL2MessageWriter as IL1ToL2MessageWriter,
+  L1ToL2MessageStatus
 } from '@arbitrum/sdk'
 import { Signer } from '@ethersproject/abstract-signer'
 import { Provider } from '@ethersproject/abstract-provider'
 import dayjs from 'dayjs'
 import { JsonRpcProvider } from '@ethersproject/providers'
+import { MergedTransaction } from '../state/app/state'
+import { isTeleport } from '../token-bridge-sdk/teleport'
 
 type GetRetryableTicketParams = {
-  l1TxHash: string
+  parentChainTxHash: string
```

Comment:
Can we rename those to `parentTxHash` (without `chain`) for all occurrences? So that `parentChain{...}` becomes `{parent{...}` - same for child. The reason for that it's the pattern in sdk v4 so it will make things more consistent in the future.